docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -198,13 +198,7 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
       rtl: showRtl,
     }
 
-    const providerVariables = showTransparent ? { background: 'initial' } : undefined
-
-    return (
-      <Provider theme={newTheme} variables={providerVariables}>
-        {element}
-      </Provider>
-    )
+    return <Provider theme={newTheme}>{element}</Provider>
```

Comment:
This breaks the possibility to render examples on transparent background:
![image](https://user-images.githubusercontent.com/9615899/57584055-7ae30600-74d7-11e9-934b-44da5684c95d.png)

Reply:
got it - the thing is that this `variables` prop was missing from the `Provider`'s API, and that has led me to confusion. Will address it